### PR TITLE
Sdk automl image object detection batch online

### PR DESCRIPTION
**Pull request by sriramya2610**
This PR includes the following updates:

1. Adds Colab Enterprise link and other structural changes from the new template.
2. Changes REGION variable name to LOCATION.
3. Hardcodes the version of tensorflow.
4. Modifies the code in cleanup section.
5. Replaces uuid with unique and removed the code of uuid generation.
6. Adds code for copying from one bucket to other for importing dataset as original dataset did not have access.

**REQUIRED:** Fill out the below checklists or remove if irrelevant
1. If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [x] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

### CODE REVIEW

File: notebooks/official/migration/sdk-automl-image-object-detection-batch-online.ipynb [hunk 1]
```diff
@@ -81,10 +81,10 @@
         "\n",
         "This tutorial uses the following Google Cloud ML services:\n",
         "\n",
-        "- `AutoML`\n",
-        "- `Vertex AI Batch Prediction`\n",
-        "- `Vertex AI Model` resource\n",
-        "- `Vertex AI Endpoint` resource\n",
+        "- AutoML\n",
+        "- Vertex AI Batch Prediction\n",
```
Comment (kittyabs) [inline notebooks/official/migration/sdk-automl-image-object-detection-batch-online.ipynb:85]: Should be: "Vertex AI Batch prediction" (i.e., lower case for "prediction")

File: notebooks/official/migration/sdk-automl-image-object-detection-batch-online.ipynb [hunk 2]
```diff
@@ -81,10 +81,10 @@
...
+        "- Vertex AI Model resource\n",
+        "- Vertex AI Endpoint resource\n",
```
Comment (kittyabs) [inline notebooks/official/migration/sdk-automl-image-object-detection-batch-online.ipynb:87]: Should be "Vertex AI endpoint resource" (i.e., lower case for "endpoint")

File: notebooks/official/migration/sdk-automl-image-object-detection-batch-online.ipynb [hunk 3]
```diff
@@ -146,288 +153,300 @@
...
       "source": [
-        "### Authenticate your Google Cloud account\n",
-        "\n",
-        "Depending on your Jupyter environment, you may have to manually authenticate. Follow the relevant instructions below.\n",
-        "\n",
-        "**1. Vertex AI Workbench**\n",
-        "* Do nothing as you are already authenticated.\n",
-        "\n",
-        "**2. Local JupyterLab instance, uncomment and run:**"
+        "**Only if your bucket doesn't already exist**: Run the following cell to create your Cloud Storage bucket."
```
Comment (kittyabs) [inline notebooks/official/migration/sdk-automl-image-object-detection-batch-online.ipynb:279]: - change: "Only if your bucket doesn't already exist"
- to: "If your bucket doesn't already exist" (i.e., remove "Only")

File: notebooks/official/migration/sdk-automl-image-object-detection-batch-online.ipynb [hunk 4]
```diff
@@ -146,288 +153,300 @@
...
-        "id": "setup_vars"
+        "id": "dd6e0b57b772"
       },
       "source": [
-        "### Set up variables\n",
+        "### Copying data between Google Cloud Storage Buckets \n",
         "\n",
-        "Next, set up some variables used throughout the tutorial.\n",
-        "### Import libraries and define constants"
+        "In this step, you prevent access issues for the images in your original dataset. The code below extracts folder paths from image paths, constructs destination paths for Google Cloud Storage (GCS), copies images using gsutil commands, updates image paths in the DataFrame, and finally saves the modified DataFrame back to GCS as a CSV file."
```
Comment (kittyabs) [inline notebooks/official/migration/sdk-automl-image-object-detection-batch-online.ipynb:368]: - change: "Google Cloud Storage (GCS),"
- to: "Cloud Storage" (i.e., remove "Google" and the abbreviation)

- change: "saves the modified DataFrame back to GCS as a CSV file."
- to: saves the modified DataFrame back to Cloud Storage as a CSV file.

File: notebooks/official/migration/sdk-automl-image-object-detection-batch-online.ipynb [hunk 5]
```diff
@@ -438,7 +457,7 @@
       "source": [
         "#### Quick peek at your data\n",
         "\n",
-        "This tutorial uses a version of the Salads dataset that is stored in a public Cloud Storage bucket, using a CSV index file.\n",
+        "This tutorial uses a version of salads dataset which is copied to the project's Cloud Storage Bucket.\n",
```
Comment (kittyabs) [inline notebooks/official/migration/sdk-automl-image-object-detection-batch-online.ipynb:460]: - change: "Cloud Storage Bucket"
- to: "Cloud Storage bucket" (i.e., change "Bucket" to lower case)

File: notebooks/official/migration/sdk-automl-image-object-detection-batch-online.ipynb [hunk 6]
```diff
@@ -1236,7 +1259,7 @@
       "source": [
         "## Undeploy the model\n",
         "\n",
-        "When you are done doing predictions, you undeploy the model from the `Endpoint` resouce. This deprovisions all compute resources and ends billing for the deployed model."
+        "When you're done doing predictions, you undeploy the model from the `Endpoint` resouce. This deprovisions all compute resources and ends billing for the deployed model."
```
Comment (kittyabs) [inline notebooks/official/migration/sdk-automl-image-object-detection-batch-online.ipynb:1262]: 1. There is a typo: "resouce" -> "resource"
2. I think "`Endpoint`" resource should be "endpoint resource"

**Review by kittyabs — COMMENTED** [review]
numerous edits

**Review by katiemn — APPROVED** [review]

**Pull request merged**